src/apps/bristleback_apps/bm_rbr/user_code/rbr_sensor_util.cpp

```diff
@@ -49,9 +49,9 @@ bool validSensorDataString(const char *s, size_t strlen) {
 bool validSensorData(DataType_e type, double val) {
   switch (type) {
   case TEMPERATURE: // in degrees C
-    return val >= -5.0 && val <= 50.0;
+    return val >= -40.0 && val <= 125.0;
   case PRESSURE: // in decibar
-    return val >= 5.0 && val <= 85.0;
+    return val >= 5.0 && val <= 200.0;
```

Comment:
These are the valid ranges for data coming from the sensor itself (as opposed to aggregated sample mean values). The sensor can read temperatures from -40 to 125 (wider than the aggregation range) and pressures with the same range as for aggregation: 0.5 bar to 20 bar.